pyro/distributions/householder.py

```diff
@@ -68,12 +69,17 @@ def __init__(self, input_dim, count_transforms=1):
                 "Number of Householder transforms, {}, is greater than input dimension {}, which is an \
 over-parametrization!".format(count_transforms, input_dim))
         self.count_transforms = count_transforms
-        self.u = nn.Parameter(torch.Tensor(count_transforms, input_dim))
+        self.u_unnormed = nn.Parameter(torch.Tensor(count_transforms, input_dim))
```

Comment:
can you make `assert count_transforms > 0` raise a `ValueError` instead?

Reply:
Is raising an exception not too extreme? It will still work when `assert count_transforms > 0`?

Reply:
i mean raise the error if that's false, e.g. if it's equal to 0 or -7

Reply:
Done!